### PR TITLE
Update windows prereqs for 2.2

### LINKED ISSUE #7941: Misleading page title?
As described in the first paragraph, the article describes the prerequisites for .NET Core development. So the title should be "Prerequisites for .NET Core development on Windows".

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff5c1c2a-7d86-703c-038e-062f5dcfc5b0
* Version Independent ID: cab4f67f-f338-8990-6454-853a0f01627e
* Content: [Prerequisites for .NET Core on Windows](https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites?tabs=netcore21)
* Content Source: [docs/core/windows-prerequisites.md](https://github.com/dotnet/docs/blob/master/docs/core/windows-prerequisites.md)
* Product: **dotnet-core**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

### LINKED ISSUE #9257: Update Prerequisites topics for .NET Core 2.2
- [ ] Windows: https://docs.microsoft.com/en-us/dotnet/core/windows-prerequisites
- [x] macOS: https://docs.microsoft.com/en-us/dotnet/core/macos-prerequisites
- [x] Linux: https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #9257 
Fixes #7941

### CODE REVIEW

File: docs/core/windows-prerequisites.md [hunk 1]
```diff
@@ -29,8 +29,12 @@ This article shows the dependencies needed to develop .NET Core applications on
...
 The following articles have a complete list of .NET Core supported operating systems per version:
 
-* [.NET Core 2.1 - Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md)
-* [.NET Core 1.x - Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0-supported-os.md)
+* [.NET Core 2.2](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md)
+* [.NET Core 2.1](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md)
+* [.NET Core 1.1](https://github.com/dotnet/core/blob/master/release-notes/1.1/1.1-supported-os.md)
+* [.NET Core 1.0](https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0-supported-os.md)
+
+For download links and more information, see [.NET downloads](https://www.microsoft.com/net/download) to download the latest version or [.NET downloads archive](https://dotnet.microsoft.com/download/archives#dotnet-core) for older versions.
```
Comment (pkulikov) [inline docs/core/windows-prerequisites.md]: @mairaw This paragraph tells the similar story as the next one. Is it intentional or a merge result?

Comment (mairaw) [reply docs/core/windows-prerequisites.md]: Nope. Good catch @pkulikov. I tried to simplify the paragraph, not duplicate it. 😄

**Comment by Thraka** [comment]
Is there a difference between 1.x and 2.x? I'm not really seeing it. Is it a specific version of Visual Studio? 2017 the first version versus 2017 latest?

**Review by Thraka — APPROVED** [review]

**Comment by mairaw** [comment]
Yes, the tab content is completely different. What happens when you click on the tab?

**Pull request merged**